docs/reference/sql/endpoints/client-apps/dbvis.asciidoc

```diff
@@ -11,34 +11,43 @@ IMPORTANT: Elastic does not endorse, promote or provide support for this applica
 * https://www.dbvis.com/[DbVisualizer] 10.0.21 or higher
```

Comment:
Since the images are for Version 13.0, how about increasing the requirement and potentially mentioning that older versions can still use ES by using the generic driver (and linking maybe to the older documentation).

Reply:
Good point, removed the "native Elasticsearch integration" recommendation. 
I've updated the requirement to 13.0 and noted that older versions will work too, but left out older doc reference for now since:
* we covered 10.0.21 and higher version, but it'll likely work with older versions too; DbVis started having an Elasticsearch driver template at some point, but a generic one could probably be used too, prior to that. And we don't have docs for that.
* we'd have to choose between a 7.x and 8.x link, which might introduce some confusion.